### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by zilton7**
The Milestone 1 for Newsweek Website Copy Project

### CODE REVIEW

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello Zil Norvilis & Francis Obetta** 
Good job with your first submission for this project :muscle:  :muscle:
Your footer and navbar meet most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:

- [x] Improve the README file, include relevant information about the project: what it does, how to set up in local environment, etc.
- [x] For mobile screen size, navbar should have a user icon

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] For mobile screen size, your footer doesn't match the structure of the original

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] Make sure your project passes all the linter checks 


Please request another review once you are ready! :+1: 
 
Happy coding :relaxed:
 
Oscar Andrés Russi Porras

File: index.html [hunk 1]
```diff
@@ -6,13 +6,130 @@
...
+  </main>
+  <footer>
+    <div class="container-fluid">
+      <nav class="navbar navbar-light justify-content-space-between align-items-center">
+        <div class="align-items-center left_side">
+          <a class="navbar-brand align-items-center bottom_nav_logo" href="#">
+            <img src="./assets/logo.svg" alt="Newsweek logo" width="300" height="30">
+          </a>
+          <div>© 2021 NEWSWEEK DIGITAL LLC</div>
+        </div>
```
Comment (SpaYco) [inline index.html]: - [x] the copyright text should be under the logo on mobile screens.

**Review by SpaYco — CHANGES_REQUESTED** [review]
# Changes Requested 

Hello Team! 👋 
Awesome job so far!! 👍 you still need to make some changes though! 

## Required Changes ♻️ 

- [ ] Fix your linter errors.

### please make the appropriate changes and ask for another code review ✔️ 

Happy Coding!

Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Review by SpaYco — APPROVED** [review]
# Project Approved ✔️ 

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**